src/viur/core/skeleton.py

```diff
@@ -1220,8 +1274,11 @@ def fixDotNames(entity):
                 updateRelations(key, time() + 1, None)
 
         # Inform the custom DB Adapter of the changes made to the entry
-        if skel.customDatabaseAdapter:
-            skel.customDatabaseAdapter.updateEntry(db_obj, skel, change_list, is_add)
+        for adapter in skel.customDatabaseAdapter:
+            adapter.updateEntry(db_obj, skel, change_list, is_add)  # FIXME: DEPRECATED: Superseeded by adapter.trigger.
```

Comment:
We decided in viur-meeting to keep both.
```suggestion
            adapter.updateEntry(db_obj, skel, change_list, is_add)
```